Mathlib/Topology/Order/Basic.lean

```diff
@@ -463,6 +463,24 @@ theorem mem_nhds_iff_exists_Ioo_subset [OrderTopology α] [NoMaxOrder α] [NoMin
     {s : Set α} : s ∈ 𝓝 a ↔ ∃ l u, a ∈ Ioo l u ∧ Ioo l u ⊆ s :=
   mem_nhds_iff_exists_Ioo_subset' (exists_lt a) (exists_gt a)
 
+theorem exists_lt_mem_of_mem_nhds [OrderTopology α] [NoMaxOrder α] [NoMinOrder α] [DenselyOrdered α]
+    {a : α} {s : Set α} (hs : s ∈ 𝓝 a) :
+    ∃ y ∈ s, y < a := by
+  rw [mem_nhds_iff_exists_Ioo_subset] at hs
+  obtain ⟨b, c, ha_mem, hbcs⟩ := hs
+  rw [mem_Ioo] at ha_mem
+  obtain ⟨y, hby, hya⟩ := exists_between ha_mem.1
+  exact ⟨y, hbcs ⟨hby, hya.trans ha_mem.2⟩, hya⟩
+
+theorem exists_gt_mem_of_mem_nhds [OrderTopology α] [NoMaxOrder α] [NoMinOrder α] [DenselyOrdered α]
```

Comment:
```suggestion
theorem exists_gt_mem_of_mem_nhds [OrderTopology α] [NoMinOrder α] [DenselyOrdered α]
```
This should still be true. Actually even the following might be true but I am not sure how to prove it
```suggestion
theorem exists_gt_mem_of_mem_nhds [OrderTopology α] [DenselyOrdered α]
```

Reply:
I removed one of the assumption. I don't think it's possible to remove the other (what if `a` is bot?).